spec/emerald/rubify_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "spec_helper"
+require "emerald/rubify"
+require "emerald/atom"
+require "emerald/string"
+require "emerald/number"
+require "emerald/list"
+
+RSpec.describe Emerald::Rubify do
+
+  it "compiles an empty source" do
+    compiled_code = Emerald::Rubify.new([]).rubify
+    expect(compiled_code).to eq('')
+  end
+
+  describe "code generation from atoms" do
+    it "generates code from a single atom" do
+      compiled_code = Emerald::Rubify.new([Emerald::Atom.new("hello-world")]).rubify
+      expect(compiled_code).to eq('hello-world')
```

Comment:
In the parser I accepted that a single atom could have '-' in between words, and even tested for it. It seems to be an acceptable way of naming atoms in lisps :)